test/Interpreter/SDK/weak_objc_interop.swift

```diff
@@ -26,3 +26,19 @@ tryWeakReferencing { C() }
 // CHECK-NEXT:  Swift Object
 // CHECK-NEXT:  after giving up strong reference:
 // CHECK-NEXT:  Gone
+
+
+// Test that instances of Swift actors can be referenced using ObjC
+// weak references.
+
+actor A {
```

Comment:
You will probably need some SwiftStdlib 5.1 availability guards on these

Reply:
Good point. I should probably move this to a separate file too so we can put `REQUIRES: concurrency` only on this test.